### DISCUSSION TITLE
Trying to create a camera similar to Super Mario World's

### DISCUSSION THREAD

**Discussion by dRoycon** [ramokz/phantom-camera#393] [Question / Help Wanted] [answered]
In mario games like Mario World, there is a dead zone like that in the phantom camera, yet the camera always follows behind the player to allow them to see ahead ([example](https://www.youtube.com/watch?v=01AHrMZ2Qvs)). I've tried creating a similar system but despite setting an offset, the camera does not show ahead of the player and always ends up behind them instead of ahead of them. Any possible fixes?

**Comment by ramokz** [comment]
This is something others have asked about too.

There is a draft PR in #284; however, I suspect that it has been abandoned by this point. So I will likely be looking at implementing a solution myself at some point post-v0.9 release.

**Comment by ramokz** [comment, marked as answer]
Resolved for 2D scenes in the 0.11 release with the addition of the Lookahead functionality.
